### PR TITLE
feat: Dropdown Component

### PR DESCRIPTION
**Pull request by kelvin3127**
Dropdown Component

### Description
A dropdown component.
Includes theme styling
- background color
- background cover hover
- on click color
- More can be added in future commits
Includes Storybook

### Testing Instructions Component

Open terminal and run "npm run dev"
Open application in browser with local host.
Import dropdown component and add to the template.
Dropdown component Props include:                                                       
            theme - Changes the theme for the dropdown.                                                                                     
If no theme are is selected component will have it's own default theme.    

### Testing Instructions Storybook

Open terminal and run "npm run storybook"
Open storybook in browser "http://localhost:6006/".
Select theme you would like to view

### Screenshots
Dropdown Component with different themes

Code 

Storybook
Outline Danger Theme

Confirm Theme

### CODE REVIEW

**Comment by roxanehe** [comment]
looks nice! well done

File: frontend/src/App.vue
Comment (YoussefLaunchUp) [inline frontend/src/App.vue:1]: This should not be in App.vue Return the file to what it was like before

**Review by YoussefLaunchUp — CHANGES_REQUESTED** [review]
The Drop down should be able to take any type of component as a child not just text through the prop array. 

image




image


You can use this as reference: https://bootstrap-vue.org/docs/components/dropdown

File: frontend/src/App.vue
Comment (kelvin3127) [reply frontend/src/App.vue:1]: Made necessary changes

**Comment by kelvin3127** [comment]
> The Drop down should be able to take any type of component as a child not just text through the prop array.
> 
> image
> 
> image
> 
> You can use this as reference: https://bootstrap-vue.org/docs/components/dropdown

Update code to fit the requirments

**Review by YoussefLaunchUp — APPROVED** [review]

**Review by roxanehe — APPROVED** [review]
good work!

**Pull request merged**